packages/website/components/button.js

```diff
@@ -82,14 +81,15 @@ export default function Button({
         'items-center',
         'justify-center',
         'px-4',
+        'md:min-h-3.25',
```

Comment:
it was not a result of your changes but i noticed the login/out button while loading on mobile doesn't have a min-height / width and looks weird; maybe you could fix while you're at it? Recommendation: min-w: 2.5rem, min-h: 2rem;

![IMG_5476](https://user-images.githubusercontent.com/43380153/146575774-e5224726-f7bd-4589-ab18-e8d7d7c0a28f.PNG)

Reply:
Great catch! Resolved!